conf/application.conf

```diff
@@ -158,7 +158,7 @@ urls {
   nactCodes = "https://www.gov.uk/guidance/national-additional-codes-for-data-element-617-of-cds"
   relevantLicenses = "https://www.gov.uk/starting-to-export/licences"
   serviceAvailability = "https://www.gov.uk/guidance/customs-declaration-service-service-availability-and-issues"
-  customsMovementsFrontend = "http://localhost:6796/customs-movements/start"
+  customsMovementsFrontend = "/customs-movements"
```

Comment:
Think this link does needs to be as before (an absolute URL). When running the service locally, if you use this relative link it will not resolve because the movements service is listening on a different port number (6796). That is why for the local app.conf we have to make it an absolute url and then override the value in app-conf-base to make it a relative url for the MDTP environments.

Reply:
:+1: